torch/csrc/utils/tensor_list.h

```diff
@@ -3,8 +3,8 @@
 #include <ATen/ATen.h>
 #include <Python.h>
 
-namespace torch {
```

Comment:
is this supposed to be part of a general pattern going forward?  i.e. prefer naming torch::utils::name rather than torch::THPUtils_name?

Reply:
Yes, I think we should use C++ namespaces instead of name prefixes like `TH`/`THP`. I think it's also good practice to have the namespace correspond to the file path.